public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -0,0 +1,110 @@
+import { css, cx } from '@emotion/css';
+import React, { SyntheticEvent, useRef, useState } from 'react';
+import Draggable from 'react-draggable';
+import { Resizable, ResizeCallbackData } from 'react-resizable';
+
+import { Dimensions2D, GrafanaTheme } from '@grafana/data';
+import { IconButton, stylesFactory, useTheme } from '@grafana/ui';
+import store from 'app/core/store';
+
+import { InlineEditOptions } from './InlineEditOptions';
+
+type Props = {
+  onClose?: () => void;
+};
+
+const OFFSET = 8;
+
+export const InlineEdit = ({ onClose }: Props) => {
+  const theme = useTheme();
+  const btnInlineEdit = document.querySelector('[data-btninlineedit]')!.getBoundingClientRect();
+  const ref = useRef<HTMLDivElement>(null);
+  const styles = getStyles(theme);
+  const inlineEditKey = 'inlineEditPanel';
+
+  const defaultMeasurements = { width: 350, height: 400 };
+  const defaultX = btnInlineEdit.x - btnInlineEdit.width + OFFSET;
+  const defaultY = -OFFSET - defaultMeasurements.height;
+
+  const savedPlacement = store.getObject(inlineEditKey, {
+    x: defaultX,
+    y: defaultY,
+    w: defaultMeasurements.width,
+    h: defaultMeasurements.height,
+  });
+  const [measurements, setMeasurements] = useState<Dimensions2D>({ width: savedPlacement.w, height: savedPlacement.h });
+  const [placement, setPlacement] = useState({ x: savedPlacement.x, y: savedPlacement.y });
+
+  const onDragStop = (event: any, dragElement: any) => {
```

Comment:
On drag stop seems to cause the inline edit window to jump (either up or down) Seems like this may be fixed via using Portal as @ryantxu mentioned below

https://user-images.githubusercontent.com/22381771/166303323-f96b5053-d272-47a3-accd-c9e1f5bd335b.mov

Reply:
Yes, it got fixed.